cmd/ekanited/main.go

```diff
@@ -70,7 +71,8 @@ func main() {
 		batchSize       = fs.Int("batchsize", DefaultBatchSize, "Indexing batch size")
 		batchTimeout    = fs.Int("batchtime", DefaultBatchTimeout, "Indexing batch timeout, in milliseconds")
 		indexMaxPending = fs.Int("maxpending", DefaultIndexMaxPending, "Maximum pending index events")
-		tcpIface        = fs.String("tcp", DefaultTCPServer, "Syslog server TCP bind address in the form host:port. To disable set to empty string")
+		tcpIfaceS       = fs.String("tcpsyslog", DefaultTCPServerS, "Syslog server TCP bind address in the form host:port. To disable set to empty string")
```

Comment:
You're right. I did this, in order to test the code, but its not a good solution.